pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -42,6 +43,62 @@ func addConversionFuncs() {
 		// If one of the conversion functions is malformed, detect it immediately.
 		panic(err)
 	}
+
+	err = api.Scheme.AddFieldLabelConversionFunc("v1beta1", "DaemonSet",
+		func(label, value string) (string, string, error) {
+			switch label {
+			case "metadata.name", "metadata.namespace":
+				return label, value, nil
+			default:
+				return "", "", fmt.Errorf("field label not supported: %s", label)
+			}
+		})
+	if err != nil {
+		// If one of the conversion functions is malformed, detect it immediately.
+		panic(err)
+	}
+
+	err = api.Scheme.AddFieldLabelConversionFunc("v1beta1", "Deployment",
```

Comment:
v1beta1 should be changed to extensions/v1beta1. Others  below should be changed  too.